packages/api/models/rules.js

```diff
@@ -1,640 +0,0 @@
-'use strict';
-
-const cloneDeep = require('lodash/cloneDeep');
-const get = require('lodash/get');
-const set = require('lodash/set');
-const merge = require('lodash/merge');
-
-const awsServices = require('@cumulus/aws-client/services');
-const CloudwatchEvents = require('@cumulus/aws-client/CloudwatchEvents');
-const log = require('@cumulus/common/log');
-const s3Utils = require('@cumulus/aws-client/S3');
-const workflows = require('@cumulus/common/workflows');
-const { invoke } = require('@cumulus/aws-client/Lambda');
-const SQS = require('@cumulus/aws-client/SQS');
-const { ValidationError } = require('@cumulus/errors');
-
-const Manager = require('./base');
-const { rule: ruleSchema } = require('./schemas');
```

Comment:
Can this schema be deleted from the `schemas.js` file?

Reply:
Sure. What could go wrong?

https://github.com/nasa/cumulus/pull/3149/commits/e85d67e3d59650e810485c08d2bc9eff21d44fc1